src/main/java/bio/terra/service/resourcemanagement/google/GoogleBucketService.java

```diff
@@ -235,12 +242,13 @@ private GoogleBucketResource createMetadataRecord(
       String flightId,
       Duration daysToLive,
       Callable<List<String>> getReaderGroups,
-      String dedicatedServiceAccount)
+      String dedicatedServiceAccount,
+      boolean autoclassEnabled)
       throws InterruptedException {
 
     // insert a new bucket_resource row and lock it
     GoogleBucketResource googleBucketResource =
-        resourceDao.createAndLockBucket(bucketName, projectResource, region, flightId);
+        resourceDao.createAndLockBucket(bucketName, projectResource, region, flightId, true);
```

Comment:
shouldn't this be
```suggestion
        resourceDao.createAndLockBucket(bucketName, projectResource, region, flightId, autoclassEnabled);
```

Reply:
Yes, you're totally right. Thank you for catching this.
I think the existence of this error shows that I don't have sufficient test coverage. I will add some alongside this fix.